week-2/debug/0.js

```diff
@@ -1,7 +1,13 @@
 // Predict and explain first...
 
 function multiply(a, b) {
-  console.log(a * b);
+  return a * b;
+  // console.log(a * b);
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+
+// The function named 'multiply' is defined to find a product of two numbers a and b. Therefore, the function will return 320
+// and the console.log in line 7 will return the following;
+// The result of multiplying 10 and 32 is undefined. This is because, calling console.log within console.log returns undefined for the inner function.
```

Comment:
You're heading the right direction here!
Try to be as precise as possible in your language here.
What does `console.log` do? And what does `console.log(a * b) return ?`
And more precisely still, what is the difference between `return a * b` and `console.log(a * b)`?